### PR TITLE
CI: docker 32-bit linux build #32709

### LINKED ISSUE #32709: Restore 32 Bit Linux CI
Noticed in #32706 and temporarily disabled in a follow up #32708

### PR DESCRIPTION
**Pull request by fangchenli**
- [x] closes #32709

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -26,3 +26,28 @@ jobs:
...
     vmImage: vs2017-win2016
+
+- job: py37_32bit
+  pool:
+    vmImage: ubuntu-18.04
+
+  steps:
+    - script: |
+        docker pull quay.io/pypa/manylinux2014_i686
+        docker run -v $(pwd):/pandas quay.io/pypa/manylinux2010_i686 \
```
Comment (WillAyd) [inline azure-pipelines.yml:37]: This is 2010 here but 2014 on the line above - do they not need to match?

File: azure-pipelines.yml [hunk 2]
```diff
@@ -26,3 +26,28 @@ jobs:
...
     name: Windows
     vmImage: vs2017-win2016
+
+- job: py37_32bit
+  pool:
+    vmImage: ubuntu-18.04
+
+  steps:
+    - script: |
+        docker pull quay.io/pypa/manylinux2014_i686
```
Comment (WillAyd) [inline azure-pipelines.yml:36]: I think we are still building manylinux1 wheels so should probably keep that in sync - @TomAugspurger

**Comment by TomAugspurger** [comment]
Thanks @VirosaLi. Are you able to integrate this into our existing setup at all? See how we handle the Python 3.9 build (e.g. https://github.com/pandas-dev/pandas/blob/master/ci/setup_env.sh#L3-L6 and elsewhere).

File: azure-pipelines.yml (hunk 1 above)
Comment (VirosaLi) [reply azure-pipelines.yml:37]: It's a typo. I'll fix it.

**Comment by jreback** [comment]
@fangchenli can you update this. would be great to get this to work (xfailing the tests as needed of course). W/o this or similar we are going to have a very hard time continuing to support 32-bit.

**Comment by simonjayhawkins** [comment]
Windows py38_np18 had stalled. restarted.

the following tests are failing on the 32bit wheel builds

```

=========================== short test summary info ===========================
FAILED test_venv/lib/site-packages/pandas/tests/groupby/test_groupby.py::test_groupby_nat_exclude
FAILED test_venv/lib/site-packages/pandas/tests/io/formats/test_info.py::test_info_int_columns
FAILED test_venv/lib/site-packages/pandas/tests/window/test_grouper.py::TestGrouperGrouping::test_groupby_rolling_string_index
FAILED test_venv/lib/site-packages/pandas/tests/window/test_grouper.py::TestGrouperGrouping::test_groupby_rolling_no_sort
= 4 failed, 64345 passed, 7420 skipped, 1047 xfailed, 6 xpassed, 14 warnings in 932.75s (0:15:32) =
```

so will want to merge some of the changes here into 1.1.x.

@jreback could wait till this is approved and merged and do a partial backport or open a separate PR now directly against 1.1.x with the changes to those tests here.

**Comment by jreback** [comment]
thanks a lot @fangchenli this is great

**Comment by jreback** [comment]
@simonjayhawkins pls backport what u need against 1.1.x

**Comment by simonjayhawkins** [comment]
@meeseeksdev backport 1.1.x

**Pull request merged**